launch-script/src/main.rs

```diff
@@ -293,6 +293,52 @@ fn main() {
                         .help("Proposal for vote")
                 )
             )
+            .subcommand(SubCommand::with_name("create-start-evm")
+                .about("Create proposal for Neon EVM start")
+                .arg(
+                    Arg::with_name("collateral-pool-base")
```

Comment:
The main idea of launch-script is to simplify work for making actions with NeonEVM. So the best practice is to calculate all necessary values if possible.
In this case, we can take `collateral-pool-base` from the maintenance governance address.

Reply:
Fixed